drivers/spi/Kconfig.stm32

```diff
@@ -32,22 +32,31 @@ config SPI_STM32_USE_HW_SS
 	help
 	  Use Slave Select pin instead of software Slave Select.
 
-config SPI_STM32F7_ERRATA_BUSY
+config SPI_STM32_ERRATA_BUSY
 	bool
 	default y
 	depends on SOC_STM32F745XX || SOC_STM32F746XX || \
-		   SOC_STM32F750XX || SOC_STM32F756XX
+		   SOC_STM32F750XX || SOC_STM32F756XX || \
+		   SOC_STM32F765XX || SOC_STM32F767XX || \
+		   SOC_STM32F769XX || SOC_STM32F722XX || \
+		   SOC_STM32F723XX || \
+		   SOC_STM32L451XX || SOC_STM32L433XX || \
+		   SOC_STM32L432XX || SOC_STM32L431XX || \
+		   SOC_STM32L496XX || SOC_STM32L4A6XX || \
+		   SOC_STM32L451XX || SOC_STM32L452XX || \
+		   SOC_STM32L462XX || SOC_STM32L412XX || \
+		   SOC_STM32L422XX || SOC_STM32L471XX
```

Comment:
I've looked into all errata sheets for these two series:
- ES0360: F72X/F73X
- ES0290: F74X/F75X
- ES0334: F76X/F77X
- ES0456: L412
- ES0457: L422
- ES0320: L431
- ES0319: L432/L442
- ES0318: L433/L443
- ES0387: L451
- ES0388: L452
- ES0389: L462
- ES0303: L471
- ES0302: L475
- ES0250: L476/L486
- ES0335: L496/L4A6
- ES0510: L4P5/L4Q5
- ES0393: L4R5/L4S5
And the chapter "BSY bit may stay high at the end of data transfer in slave mode" is always present.
Unless I'm missing something, I think this could be summarized in `depends on SOC_SERIES_STM32F7X || SOC_SERIES_STM32L4X`.

Reply:
I'm fine with `depends on SOC_SERIES_STM32F7X || SOC_SERIES_STM32L4X` as well. This boils down to the question if it is better to have another MCU without the erratum, but for which the workaround is incorrectly active, or to have a MCU with the erratum for which the workaround is inactive.

What are the chances for ST to develop another F7 or L4 MCU which won't have this erratum?

Reply:
> What are the chances for ST to develop another F7 or L4 MCU which won't have this erratum?

I think we're safe on this front.